src/server/db/schema/tennant.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  decimal,
+  integer,
+  pgTable,
+  serial,
+  varchar,
+} from "drizzle-orm/pg-core";
+import { doctor } from "./doctor";
+import { relations } from "drizzle-orm";
+
+export const tennant = pgTable("tennant", {
+  id: serial("id").primaryKey(),
+  name: varchar("name").notNull(),
+  basis: decimal("basis").notNull(),
+
+  pzsName: varchar("pzs_name").notNull(),
+  regionalAuthority: varchar("regional_authority").notNull(),
+
+  pzsID: integer("pzs_id").notNull(),
```

Comment:
Remove pzs related stuff - mpve to separate table called Tenant Healthcare Provider

Reply:
Add HeadDoctor column